-code-checks

Logs: https://cmssdt.cern.ch/SDT/code-checks/cms-sw-PR-47921/44574

ERROR: Build errors found during clang-tidy run.
```
src/Geometry/HGCalGeometry/src/HGCalGeometry.cc:125:11: error: use of undeclared identifier 'x' [clang-diagnostic-error]
  125 |                                     <<x " ERROR ???";
      |                                       ^
src/Geometry/HGCalGeometry/src/HGCalGeometry.cc:125:12: error: expected ';' after expression [clang-diagnostic-error]
  125 |                                     <<x " ERROR ???";
      |                                        ^
      |                                        ;
src/Geometry/HGCalGeometry/src/HGCalGeometry.cc:125:13: error: expression result unused [clang-diagnostic-unused-value]
  125 |                                     <<x " ERROR ???";
      |                                         ^~~~~~~~~~~~
Suppressed 712 warnings (712 in non-user code).
--
gmake: *** [config/SCRAM/GMake/Makefile.coderules:129: code-checks] Error 2
gmake: *** [There are compilation/build errors. Please see the detail log above.] Error 2
```

--- a/Geometry/HGCalCommonData/src/HGCalDDDConstants.cc
+++ b/Geometry/HGCalCommonData/src/HGCalDDDConstants.cc
@@ -785,12 +785,12 @@ bool HGCalDDDConstants::isValidTrap(int zside, int layer, int irad, int iphi) co
   bool ok = ((hgpar_->scintValidRing(indx.first, irad)) && (iphi > 0) && (iphi <= hgpar_->scintCells(layer)));
   bool valid = ((ok && trapezoidFile()) ? tileExist(zside, layer, irad, iphi) : ok);
   bool tileEx = trapezoidFile() ? tileExist(zside, layer, irad, iphi) : true;
+#ifdef EDM_ML_DEBUG
   if (!valid)
     edm::LogVerbatim("HGCalGeomT") << "HGCalDDDConstants::isValidityTrap: Input " << zside << ":" << layer << ":" << irad
 				   << ":" << iphi << " Range on Ring " << hgpar_->scintValidRing(indx.first, irad)
 				   << " Range on phi 0:" << hgpar_->scintCells(layer) << " tileExist " << tileEx
 				   << " Valid " << ok << ":" << tileExist(zside, layer, irad, iphi) << ":" << valid;
-#ifdef EDM_ML_DEBUG
   else
     edm::LogVerbatim("HGCalGeomT") << "HGCalDDDConstants::isValidityTrap: Input " << zside << ":" << layer << ":"
                                    << irad << ":" << iphi << " Range on Ring "

--- a/Geometry/HGCalCommonData/src/HGCalDDDConstants.cc
+++ b/Geometry/HGCalCommonData/src/HGCalDDDConstants.cc
@@ -784,8 +784,8 @@ bool HGCalDDDConstants::isValidTrap(int zside, int layer, int irad, int iphi) co
     return false;
   bool ok = ((hgpar_->scintValidRing(indx.first, irad)) && (iphi > 0) && (iphi <= hgpar_->scintCells(layer)));
   bool valid = ((ok && trapezoidFile()) ? tileExist(zside, layer, irad, iphi) : ok);
-  bool tileEx = trapezoidFile() ? tileExist(zside, layer, irad, iphi) : true;
 #ifdef EDM_ML_DEBUG
+  bool tileEx = trapezoidFile() ? tileExist(zside, layer, irad, iphi) : true;
   if (!valid)
     edm::LogVerbatim("HGCalGeomT") << "HGCalDDDConstants::isValidityTrap: Input " << zside << ":" << layer << ":"
                                    << irad << ":" << iphi << " Range on Ring "

--- a/Geometry/HGCalCommonData/src/HGCalDDDConstants.cc
+++ b/Geometry/HGCalCommonData/src/HGCalDDDConstants.cc
@@ -787,10 +787,11 @@ bool HGCalDDDConstants::isValidTrap(int zside, int layer, int irad, int iphi) co
   bool tileEx = trapezoidFile() ? tileExist(zside, layer, irad, iphi) : true;
 #ifdef EDM_ML_DEBUG
   if (!valid)
-    edm::LogVerbatim("HGCalGeomT") << "HGCalDDDConstants::isValidityTrap: Input " << zside << ":" << layer << ":" << irad
-				   << ":" << iphi << " Range on Ring " << hgpar_->scintValidRing(indx.first, irad)
-				   << " Range on phi 0:" << hgpar_->scintCells(layer) << " tileExist " << tileEx
-				   << " Valid " << ok << ":" << tileExist(zside, layer, irad, iphi) << ":" << valid;
+    edm::LogVerbatim("HGCalGeomT") << "HGCalDDDConstants::isValidityTrap: Input " << zside << ":" << layer << ":"
+                                   << irad << ":" << iphi << " Range on Ring "
+                                   << hgpar_->scintValidRing(indx.first, irad)
+                                   << " Range on phi 0:" << hgpar_->scintCells(layer) << " tileExist " << tileEx
+                                   << " Valid " << ok << ":" << tileExist(zside, layer, irad, iphi) << ":" << valid;
   else
     edm::LogVerbatim("HGCalGeomT") << "HGCalDDDConstants::isValidityTrap: Input " << zside << ":" << layer << ":"
                                    << irad << ":" << iphi << " Range on Ring "
@@ -1475,8 +1476,8 @@ bool HGCalDDDConstants::tileExist(int zside, int layer, int ring, int phi) const
     ok = (itr == hgpar_->tileInfoMap_.end()) ? false : HGCalTileIndex::tileFineExist(itr->second.hex, zside, phi);
     if (!ok)
       edm::LogVerbatim("HGCalGeomT") << "TileExist:input " << zside << ":" << layer << ":" << ring << ":" << phi
-				     << " Index flag " << indx << ":" << (itr != hgpar_->tileInfoMap_.end()) << " ok "
-				     << ok;
+                                     << " Index flag " << indx << ":" << (itr != hgpar_->tileInfoMap_.end()) << " ok "
+                                     << ok;
     return ok;
   } else {
     int indx = HGCalTileIndex::tileIndex(layer, ring, 0);
@@ -1484,8 +1485,8 @@ bool HGCalDDDConstants::tileExist(int zside, int layer, int ring, int phi) const
     ok = (itr == hgpar_->tileInfoMap_.end()) ? false : HGCalTileIndex::tileExist(itr->second.hex, zside, phi);
     if (!ok)
       edm::LogVerbatim("HGCalGeomT") << "TileExist:input " << zside << ":" << layer << ":" << ring << ":" << phi
-				     << " Index flag " << indx << ":" << (itr != hgpar_->tileInfoMap_.end()) << " ok "
-				     << ok;
+                                     << " Index flag " << indx << ":" << (itr != hgpar_->tileInfoMap_.end()) << " ok "
+                                     << ok;
     return ok;
   }
   return ok;

--- a/Geometry/HGCalCommonData/src/HGCalTileIndex.cc
+++ b/Geometry/HGCalCommonData/src/HGCalTileIndex.cc
@@ -96,8 +96,8 @@ bool HGCalTileIndex::tileFineExist(const int32_t* hex, int32_t zside, int32_t ip
   bool ok = (hex[iw] & (1 << ibit));
   if (!ok)
     edm::LogVerbatim("HGCalGeom") << "tileFineExist::Phi " << iphi << " " << phi << " zside " << zside << " jj|iw|ibit "
-				  << jj << ":" << iw << ":" << ibit << " hex " << std::hex << hex[iw] << ":"
-				  << (1 << ibit) << std::dec << " OK " << ok;
+                                  << jj << ":" << iw << ":" << ibit << " hex " << std::hex << hex[iw] << ":"
+                                  << (1 << ibit) << std::dec << " OK " << ok;
 #ifdef EDM_ML_DEBUG
   else
     edm::LogVerbatim("HGCalGeom") << "tileFineExist::Phi " << iphi << " " << phi << " zside " << zside << " jj|iw|ibit "

--- a/Geometry/HGCalGeometry/src/HGCalGeometry.cc
+++ b/Geometry/HGCalGeometry/src/HGCalGeometry.cc
@@ -122,7 +122,7 @@ void HGCalGeometry::newCell(
       edm::LogVerbatim("HGCalGeom") << "Check " << HGCScintillatorDetId(idc) << " from " << HGCScintillatorDetId(detId)
 				    << " Mode " << m_topology.dddConstants().geomMode() << ":" << m_topology.geomMode()
 				    << " Valid " << m_topology.tileTrapezoid() << ":" << m_topology.valid(idc)
-				    <<x " ERROR ???";
+				    << " ERROR ???";
     }
   } else {
 #ifdef EDM_ML_DEBUG

--- a/Geometry/HGCalGeometry/src/HGCalGeometry.cc
+++ b/Geometry/HGCalGeometry/src/HGCalGeometry.cc
@@ -120,9 +120,9 @@ void HGCalGeometry::newCell(
 #endif
     } else {
       edm::LogVerbatim("HGCalGeom") << "Check " << HGCScintillatorDetId(idc) << " from " << HGCScintillatorDetId(detId)
-				    << " Mode " << m_topology.dddConstants().geomMode() << ":" << m_topology.geomMode()
-				    << " Valid " << m_topology.tileTrapezoid() << ":" << m_topology.valid(idc)
-				    << " ERROR ???";
+                                    << " Mode " << m_topology.dddConstants().geomMode() << ":" << m_topology.geomMode()
+                                    << " Valid " << m_topology.tileTrapezoid() << ":" << m_topology.valid(idc)
+                                    << " ERROR ???";
     }
   } else {
 #ifdef EDM_ML_DEBUG

